tests/test_logging.py

```diff
@@ -60,6 +60,7 @@ def test_logging(self):
             simulator.setup_simulated_entities(override_dict)
             simulator.run_simulation()
             self.assertLogs(level=level)
+            handler.close()
```

Comment:
Why did you need to add this? I think I was getting errors from not having it, but I'm curious what lead you to figure it out?

Reply:
it gives an error on windows

Reply:
the error is
```
>       os.remove(file_loc)
E       PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\code\\elfpy\\.logging\\test_sim.log'
```